packages/client/hmi-client/src/components/Navbar.vue

```diff
@@ -238,7 +258,46 @@ i {
 	margin-right: 0.5rem;
 }
 
-.logo {
-	align-self: center;
+/* Suggested terms */
+.suggested-terms {
+	align-items: center;
+	display: flex;
+	column-gap: 0.5rem;
+	grid-area: suggested-terms;
+	justify-content: center;
+	margin-top: 0.5rem;
+	overflow: hidden;
+	white-space: nowrap;
+}
+
+.suggested-terms,
+.p-chip {
+	font-size: small;
+	font-weight: bold;
+	color: var(--text-color-subdued);
+}
+
```

Comment:
Not sure how `p-chip` styles are back in here. Remove them and for the above style move those into just `.suggested terms`